core/tools/arxiv/arxiv_metadata_config.py

```
@@ -0,0 +1,275 @@
+
+        Returns:
+            Configuration dictionary
+        """
+        return {
+            "name": "arxiv_metadata_workflow",
+            "batch_size": self.batch_size,
+            "num_workers": 1,  # Single-threaded for streaming
+            "use_gpu": self.use_gpu,
+            "checkpoint_enabled": self.resume_from_checkpoint,
+            "checkpoint_interval": self.checkpoint_interval,
+            "staging_path": Path("/tmp/arxiv_metadata_staging"),
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/arxiv_metadata_staging"

(S108)